app/components/agenda/agendaitem/agendaitem-postponed.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import addWeeks from 'date-fns/addWeeks';
+import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+
+/**
+ * @argument subcase
+ * @argument meeting
+ * @argument agendaActivity
+ */
+export default class AgendaitemPostponed extends Component {
+  @service store;
+  @service agendaService;
+
+  @tracked canPropose = false;
+  @tracked modelsForProposedAgenda;
+  @tracked newMeeting;
+
+  constructor() {
+    super(...arguments);
+    this.loadProposedStatus.perform();
+    this.loadProposableMeetings.perform();
+  }
+
+  @task
+  *loadProposedStatus() {
+    // We have to check if the proposed agendaitem is already proposed for a new meeting before showing the button
+    // we have to generate a link to the latest meeting (not the next if multiple).
+    // filtering on meeting.plannedStart will not find any if reProposing happened before the meeting "started" at 10 am
+    let agendaActivities = yield this.store.query('agenda-activity', {
+      'filter[subcase][:id:]': this.args.subcase.id,
+      'filter[:gt:start-date]':
+        this.args.agendaActivity.startDate.toISOString(),
+      sort: '-start-date',
+    });
+    // If any agenda-activities exist that are created after this one we can assume the subcase is already proposed again.
+    this.canPropose = agendaActivities.length ? false : true;
+    if (!this.canPropose) {
+      // load the latest agenda link
+      const latestActivity = agendaActivities.firstObject;
+      const latestAgendaitem = yield this.store.queryOne('agendaitem', {
+        'filter[agenda-activity][:id:]': latestActivity.id,
+        'filter[:has-no:next-version]': 't',
+        sort: '-created',
+      });
+      const agenda = yield latestAgendaitem.agenda;
+      const meeting = yield agenda.createdFor;
+      this.newMeeting = meeting;
```

Comment:
`this.proposedMeeting` instead of `this.newMeeting`?

Reply:
We are showing the latest meeting connected to subcase.
I guess `this.latestMeeting` or something would make more sense.